tests/test_ast.py

```diff
@@ -0,0 +1,24 @@
+import nose.tools
+
+import claripy
+
+
+def test_lite_repr():
+    one = claripy.BVV(1, 8)
+    two = claripy.BVV(2, 8)
+    a = claripy.BVS('a', 8)
+    b = claripy.BVS('b', 8)
```

Comment:
you can avoid the regex assert (please avoid the regex assert) by either templating the target string or using `explicit_name=True` in the BVS constructor.

Reply:
I didn't know about `explicit_name=True` use. Thanks for the information!
I've just made the change.